src/vs/workbench/contrib/testing/browser/testingOutputPeek.ts

```diff
@@ -92,18 +92,18 @@ export class TestingPeekOpener extends Disposable implements ITestingPeekOpener
 	private lastUri?: TestUriWithDocument;
 
 	/** @inheritdoc */
-	public readonly historyVisible = MutableObservableValue.stored(this._register(new StoredValue<boolean>({
+	public readonly historyVisible = this._register(MutableObservableValue.stored(new StoredValue<boolean>({
```

Comment:
In these cases, `MutableObservableValue.stored` registers the argument so I don't think we need to registered it again. I believe the `MutableObservableValue` should then be disposed of instead